src/pygama/evt/build_evt.py

```diff
@@ -0,0 +1,1567 @@
+                        "aggregation_mode": "any",
+                        "expression": "dsp.wf_max>a",
+                        "parameters": {"a":15100},
+                        "initial": false
+                    },
+                    "multiplicity":{
+                        "channels":  ["geds_on","geds_no_psd","geds_ac"],
+                        "aggregation_mode": "sum",
+                        "expression": "hit.cuspEmax_ctc_cal > a",
+                        "parameters": {"a":25},
+                        "initial": 0
+                    },
+                    "t0":{
+                        "aggregation_mode": "keep_at:evt.energy_id",
+                        "expression": "dsp.tp_0_est"
+                    },
+                    "lar_energy":{
+                        "channels": "spms_on",
+                        "aggregation_mode": "function",
+                        "expression": ".modules.spm.get_energy(0.5,evt.t0,48000,1000,5000)"
+                    },
+                }
+            }
+
+    wo_mode
+        writing mode
+    group
+        lh5 root group name
+    tcm_group
+        lh5 root group in tcm file
+    """
+
+    lstore = store.LH5Store()
+    tbl_cfg = evt_config
+    if not isinstance(tbl_cfg, (str, dict)):
+        raise TypeError()
+    if isinstance(tbl_cfg, str):
+        with open(tbl_cfg) as f:
+            tbl_cfg = json.load(f)
+
+    if "channels" not in tbl_cfg.keys():
+        raise ValueError("channel field needs to be specified in the config")
+    if "operations" not in tbl_cfg.keys():
+        raise ValueError("operations field needs to be specified in the config")
+
+    # create channel list according to config
+    # This can be either read from the meta data
+    # or a list of channel names
+    log.debug("Creating channel dictionary")
+
+    chns = {}
+
+    for k, v in tbl_cfg["channels"].items():
+        if isinstance(v, dict):
+            # it is a meta module. module_name must exist
+            if "module" not in v.keys():
+                raise ValueError(
+                    "Need module_name to load channel via a meta data module"
+                )
+
+            attr = {}
+            # the time_key argument is set to the time key of the DSP file
+            # in case it is not provided by the config
+            if "time_key" not in v.keys():
+                attr["time_key"] = re.search(r"\d{8}T\d{6}Z", f_dsp).group(0)
+
+            # if "None" do None
+            elif "None" == v["time_key"]:
+                attr["time_key"] = None
+
+            # load module
+            p, m = v["module"].rsplit(".", 1)
+            met = getattr(import_module(p, package=__package__), m)
+            chns[k] = met(v | attr)
+
+        elif isinstance(v, str):
+            chns[k] = [v]
+
+        elif isinstance(v, list):
+            chns[k] = [e for e in v]
+
+    nrows = len(
+        store.load_nda(f_tcm, ["cumulative_length"], tcm_group)["cumulative_length"]
```

Comment:
Assuming you have `store = LH5Store()` somewhere above:
```suggestion
        store.read(f_tcm, ["cumulative_length"], tcm_group).view_as("np")
```